### PR TITLE
MAT-4530

### PR DESCRIPTION
**Pull request by praveenavas**
MAT-4530

### CODE REVIEW

File: cypress/config/dev.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "baseUrl": "https://dev-madie.hcqis.org",
   "screenshotsFolder": "mochawesome-report/assets",
-  "numTestsKeptInMemory": 0,
+  "numTestsKeptInMemory": 20,
```
Comment (cvasile001) [inline cypress/config/dev.json:4]: why are you changing this? We had put this to 0 for the memory issues we were having in Jenkins a while back.

**Review by cvasile001 — CHANGES_REQUESTED** [review]
Left one comment about the numTestsKeptInMemory

File: cypress/config/dev.json (hunk 1 above)
Comment (praveenavas) [reply cypress/config/dev.json:4]: oh sorry I changed it in my local for debugging purpose & pushed it by accident. Will revert it

**Review by cvasile001 — APPROVED** [review]
looks good

**Pull request merged**